src/resources/MachineLearning/ModelDetailedInfo/ModelDetailedInfoInterfaces.ts

```diff
@@ -35,12 +35,7 @@ export interface ModelIssues {
     troubleshoot: string;
 }
 
-export interface ModelStatusInfo {
-    /**
-     * The status of the model.
-     * @example ACTIVE
-     */
-    modelStatus: MLModelStatus;
+export interface ModelStatusInfo extends Omit<MLModelStatusInfo, 'daysUntilArchival'> {
```

Comment:
Instead of redefining `daysUntilArchival` here you should leverage `Required` https://www.typescriptlang.org/docs/handbook/utility-types.html#requiredtype

```ts
export type ModelStatusInfo = Required<Pick<MLModelStatusInfo, 'modelStatus' | 'daysUntilArchival'>>;
```